paxml/contrib/gpu/scripts_gpu/tfds_pile.py

```diff
@@ -1,15 +1,186 @@
-# coding=utf-8
```

Comment:
can you keep the header and just add the additional comment instead? otherwise import is broken